docs/installation.mdx

```diff
@@ -123,4 +123,42 @@ Version:        1.0.0
 Go version:     go1.20.2
 Git commit:     b58e7b910ca556973d111e9bd734a71baef03db2
 Git tree state: clean
+```
+
+## To Validate the Checksum of ORAS CLI Binaries
+
+
+The following commands can be used to verify the ORAS CLI binaries using GPG:
+
+1. First, we import a GPG Key which can be used for verification:
+
+```
+$ curl -sSL https://github.com/qweeah.gpg | gpg --import -
+gpg: key 276D8A724CE1C704: "Billy Zha <jinzha1@microsoft.com>" not changed
+gpg: Total number processed: 1
+gpg: unchanged: 1
+```
+
```

Comment:
I think we don't need to sign and trust the key, just step 1 and 4 are already enough.

Reply:
@qweeah Thank you, is it possible to get an `.asc` file without signing it? Won't we need that for verification? For me, it works without step 3 too.

Reply:
>  is it possible to get an .asc file without signing it?

Yes, it's in the release. E.g. in https://github.com/oras-project/oras/releases/tag/v1.0.0, you may find files ending with `.asc`, which are generated during releasing.

![image](https://github.com/oras-project/oras-www/assets/4254263/f6849d29-8770-4ba6-bf2c-13fdd5ceaee9)

Reply:
You may explain where to download the `.asc` files if you think it's worth mentioning.

Reply:
Sorry I had missed that, thank you!